api/types.go

```diff
@@ -88,6 +102,23 @@ type Runner struct {
 	NumThread          int     `json:"num_thread,omitempty"`
 }
 
+type PredictResponse struct {
+	Model     string    `json:"model"`
+	CreatedAt time.Time `json:"created_at"`
+	Response  string    `json:"response,omitempty"` // the latest response chunk when streaming from /generate
```

Comment:
Let's keep these structs separated, merging them (and needing comments) is more confusing – it's fine if they share another struct like `Timings` (choose a better name :)